pkg/util/util.go

```diff
@@ -69,6 +69,14 @@ func logPanic(r interface{}) {
 	glog.Errorf("Recovered from panic: %#v (%v)\n%v", r, r, callers)
 }
 
+// Min will return the smaller of the 2 given integers.
```

Comment:
Recommend just writing the if statement everywhere instead of making this function. It's unfortunate, but that's what the language encourages, since you'd have to make a different function for every numeric type.